idunn/blocks/wikipedia.py

```diff
@@ -111,6 +156,25 @@ class WikipediaBlock(BaseBlock):
 
     @classmethod
     def from_es(cls, es_poi, lang):
+        """
+        If "wikidata_id" is present and "lang" is in "ES_WIKI_LANG",
+        then we try to fetch our "WIKI_ES",
+        else then we fetch the wikipedia API.
+        """
+        wikidata_id = es_poi.get("properties", {}).get("wikidata")
+        if wikidata_id is not None:
+            wiki_index = Wikidata.get_wiki_index(lang)
+            if wiki_index is not None:
+                wiki_poi_info = Wikidata.get_wiki_info(wikidata_id, lang, wiki_index)
+                if wiki_poi_info is not None:
+                    return cls(
+                        url=wiki_poi_info[0],
+                        title=wiki_poi_info[1][0],
+                        description=wiki_poi_info[2],
+                    )
+                else:
+                    return None
```

Comment:
I would be usefull to explain why we return here

and no need to nest it in a else since the other path returns too

Reply:
done